frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java

```diff
@@ -11,17 +11,20 @@ public static Builder builder() {
     return new AutoValue_PlaceVisitModel.Builder().setUserMark(UserMark.UNKNOWN);
   }
 
+  @Nullable public abstract String uuid();
```

Comment:
Nit: you could call this `id`, since clients should be agnostic to how the ID itself is represented/created. It's just an opaque string. That would also match the current naming convention in `TripModel.`

If you do change this, don't forget to update the type adapter to match.